src/helpers/getPetitionsQueryParams.js

```diff
@@ -4,5 +4,5 @@ export default (params = {}, query = {}) => ({
   cityName: params.cityName || '',
   limit: parseInt(query.limit || 12),
   state: query.state || 'current',
-  sort: query.sort || ''
+  sort: query['sort'] || ''
```

Comment:
My syntax highlighting was showing that `query.sort` was corresponding to the native `sort()` method, so I was OCD and changed it :P